packages/zend-xml/library/Zend/Xml/Security.php

```diff
@@ -83,7 +96,7 @@ public static function scan($xml, DOMDocument $dom = null)
         }
 
         if (!self::isPhpFpm()) {
-            $loadEntities = libxml_disable_entity_loader(true);
+            $loadEntities = self::disableEntityLoader(true);
```

Comment:
@leendt is it necessary to pass `true` since it's a default value?

Reply:
Besides that, it looks fine.

Reply:
No it's not necessary. It was not necessary for the original PHP function either:
```php
/**
 * Disable the ability to load external entities
 * @link https://php.net/manual/en/function.libxml-disable-entity-loader.php
 * @param bool $disable [optional] <p>
 * Disable (<b>TRUE</b>) or enable (<b>FALSE</b>) libxml extensions (such as
 * ,
 * and ) to load external entities.
 * </p>
 * @return bool the previous value.
 * @since 5.2.11
 */
function libxml_disable_entity_loader ($disable = true) {}
```

The original Zend code probably passed it for readability/clarity.

I've copied the `libxml_disable_entity_loader` function signature for the same reason but I admit that's a weak one since it's a private method where the bool is always passed anyway. 
We can either remove the default value from the wrapper or stop passing a value.

Should I change it?

Reply:
you need to pass the value when the restore happens for libxml < 2.9.0, I'll say leave as is.